components/tikv_util/src/yatp_pool/mod.rs

```diff
@@ -295,13 +364,21 @@ impl<T: PoolTicker> YatpPoolBuilder<T> {
             .name_prefix
             .clone()
             .unwrap_or_else(|| "yatp_pool".to_string());
+        let multilevel_builder = multilevel::Builder::new(
+            multilevel::Config::default()
+                .name(Some(name))
+                .cleanup_interval(self.cleanup_method.preferred_interval()),
+        );
+        let cleanup_task = self.try_spawn_cleanup(multilevel_builder.cleanup_fn());
```

Comment:
The code here seems a bit difficult to understand at first. I prefer to comment that `cleanup_task` being `Some` means it's a task that should be spawned to the pool we are creating, otherwise it's either unnecessary or already spawned to a remote pool.

Reply:
I've added some comments for `CleanupMethod::try_spawn` and renamed `cleanup_task` here to `pending_task`. Is it more clearer now?